terraform/ec2-instance.tf

```diff
@@ -1,39 +1,39 @@
-# resource "aws_instance" "myec2vm" {
-#   ami           = data.aws_ami.amzlinux2.id
-# #   instance_type = var.instance_type
-# #   instance_type = var.instance_type_list[1]
-#   instance_type = var.instance_type_map["dev"]
-#   user_data     = file("${path.module}/app1-install.sh")
-#   key_name      = var.instance_keypair
-#   vpc_security_group_ids = [
-#     aws_security_group.vpc-ssh.id,
-#     aws_security_group.vpc-web.id
-#   ]
-#   count = 1
-#   tags = {
-#     "Name" = "${var.Name}_flow_${count.index}"
-#     Service = "CICD"
-#     Environment = "GithubActionsWorkflow"
-#   }
-#   provider = aws.us-east-1
-# }
+resource "aws_instance" "myec2vm" {
+  ami           = data.aws_ami.amzlinux2.id
+#   instance_type = var.instance_type
+#   instance_type = var.instance_type_list[1]
+  instance_type = var.instance_type_map["dev"]
```

Comment:
We want a type QA instance

Reply:
instance_type = var.instance_type_map["qa"]